components/centraldashboard/public/components/manage-users-view.js

```diff
@@ -0,0 +1,90 @@
+import '@polymer/iron-ajax/iron-ajax.js';
+import '@polymer/iron-icon/iron-icon.js';
+import '@polymer/iron-icons/iron-icons.js';
+import '@polymer/iron-icons/social-icons.js';
+import '@polymer/paper-toast/paper-toast.js';
+import '@polymer/paper-ripple/paper-ripple.js';
+import '@polymer/paper-item/paper-icon-item.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@vaadin/vaadin-grid/vaadin-grid.js';
+import '@vaadin/vaadin-grid/vaadin-grid-selection-column.js';
+import '@vaadin/vaadin-grid/vaadin-grid-sort-column.js';
+
+import {html, PolymerElement} from '@polymer/polymer';
+
+import './resources/paper-chip.js';
+import css from './manage-users-view.css';
+import template from './manage-users-view.pug';
+import utilitiesMixin from './utilities-mixin.js';
+
+export class ManageUsersView extends utilitiesMixin(PolymerElement) {
+    static get template() {
+        return html([`
+            <style>${css.toString()}</style>
+            ${template()}
+        `]);
+    }
+
+    /**
+     * Object describing property-related metadata used by Polymer features
+     */
+    static get properties() {
+        return {
+            user: {type: String, value: 'Loading...'},
+            isClusterAdmin: {type: Boolean, value: false},
+            namespaces: Array,
+            ownedNamespace: {type: Object, value: () => ({})},
+            newContribEmail: String,
+            contribError: Object,
+            contributorInputEl: Object,
+        };
+    }
+    ready() {
+        super.ready();
+        this.contributorInputEl = this.$.ContribEmail;
+    }
+    nsBreakdown(ns) {
```

Comment:
Can you add doc block comments to these methods to describe their behavior? Also, it might be worth throwing some of the namespace transformation code that seems common between a few places into a shared module somewhere.

Reply:
Done. There is no transformation that's being repeated. And the one, in this case, is simply excluding from the list of all namespaces the `ownedNamespace` which was pre-computed in main-app